.github/workflows/mt.yaml

```diff
@@ -1,7 +1,16 @@
 # yamllint disable rule:line-length
 # yamllint disable rule:braces
 
-name: Mutation Testing
+# This build is to test the whole execute the PHPUnit tests & collect the coverage
+# report and run infection.
+# We are not interested in:
+# - Running it on different systems (e.g. Windows, which was disabled because too slow)
+# - Min/maxing the PHP version used: it can be any supported version
+# - Using different dependencies than the locked ones
+# - Using different code coverages
+# All of those variants may be interesting but are better tested in more scoped
+# tests like in ci.yaml.
+name: Infection on Infection
```

Comment:
Let's revert to `Mutation Testing` as now the full name is a bit strange:

`Infection on Infection / Infection complete run on Infection`

After reverting, will be:

`Mutation Testing / Infection complete run on Infection`